pkg/common/volume_operations.go

```diff
@@ -241,7 +241,12 @@ func (vo *VolumeOperationsImpl) handleVolumeCreation(ctx context.Context, log *l
 	vo.cache.Set(v.Id, podNamespace)
 
 	// decrease AC size
-	ac.Spec.Size -= allocatedBytes
+        if util.IsStorageClassLVG(sc) {
+                ac.Spec.Size -= allocatedBytes
```

Comment:
considering such scenario,  LVG group size is not MB align,  then it seems we can never used up the LVG ?  ac.Spec.Size will somehow not zero, right?

Reply:
Yes, you are right.  LVG group ac.Spec.Size  is not MB aligned, while allocatedBytes is 4MB aligned 
https://github.com/dell/csi-baremetal/blob/master/pkg/common/volume_operations.go#L206